packages/composer/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
add types

Reply:
added types in tsconfig.json, please re-review.